src/test/resources/test.conf

```diff
@@ -1,4 +1,8 @@
-docker.file = "src/test/resources/docker-db.yml"
+docker.file = [
+    "src/test/resources/docker-mongo-latest.yml",
```

Comment:
nope, we want known versions here.

Reply:
Version wird ja in docker-db überschrieben. Aber ist vielleicht zu viel des Guten --> Rolled back